version1/02 - Object types.md

```diff
@@ -90,863 +94,615 @@ The following notation is used in the tables below:
 - **Unlink**
     - This field can be used in Unlink operations. (ref* fields)
 
+### User permissions
+**Permissions** - required user rigts/permissions to carry on update operations
+- **USM**
+    - Update System Managed - allows modification of special system fields
+- **EFO**
+    - Edit Finalized Objects - allows modification of finalized objects
+- **EJO**
+    - Edit Journaled Objects - allows modification of journaled objects
```

Comment:
Only Registry Entries (Journalpost) can be journaled and require that permission.